examples/v3.1/webhook-example.yaml

```diff
@@ -0,0 +1,61 @@
+openapi: 3.1.0
+info:
+  title: Webhook Example
+  version: 1.0.0
+paths:
+  # OpenAPI specs all need a paths element
```

Comment:
documents for specs, but we can fix this later.

Reply:
Fixed!